test_rmw_implementation/test/test_serialize_deserialize.cpp

```diff
@@ -37,6 +37,12 @@ class CLASSNAME (TestSerializeDeserialize, RMW_IMPLEMENTATION) : public ::testin
 {
 };
 
+TEST_F(CLASSNAME(TestSerializeDeserialize, RMW_IMPLEMENTATION), get_serialization_format) {
+  const char * serialization_format = rmw_get_serialization_format();
+  EXPECT_NE(nullptr, serialization_format);
+  EXPECT_STREQ(serialization_format, rmw_get_serialization_format());
```

Comment:
You're checking `rmw_get_serialization_format()` equals itself?

Reply:
Checking that the outcome is not null, and that two subsequent calls yield the same outcome -- a poor man's _is this constant_ predicate. I don't feel strongly about it.

Reply:
I think if you're trying to ensure that the format string is identical, you should be asking the question why would it change? Would you be worried it is it based on some clock-time? Then it's unlikely to change in two lines of code. This test only seems to catch the case it would change due to some random internal state, which seems strange?

I think you should add a comment describing why this is necessary.